_posts/2022-09-05-edge-example-app-sense-the-internet-part-2.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title:  "Edge Example App: Sense the Internet Part 2"
+date:   2022-09-05 03:27:05 +0100
+categories: flotta
+author: Ahmad Ateya <ahmad.m.ateya@gmail.com>
+tags:
+- flotta
+- devices
+- workloads
+- gsoc
+
+summary-1:
+---
+Edge Example App is an app for Flotta Edge devices, with a workload that will be deployed on the device that has two main features:
+- Sensing the Internet (which helps to construct devices network topology). <= this article (Part 2)
+- Read CPU temperature (which indicate how much load the device is handling).
+
+## How this app works
+This app is the presenting layer of the Flotta Edge devices, get its data from the backend app endpoints and display it to the user.
+
+you can list all devices that uses your S3 Bucket as storage bucket, and for each device you can see its information and its network topology.
+
+Here are screenshots of the app:
+
+- List Devices page
+
+<img src="/assets/images/list-devices-page.png" alt="List devices" width="800"/>
+
+- Device page with app features
+
+<img src="/assets/images/device-features-page.png" alt="List devices" width="800"/>
+
+- Network topology page
+
+<img src="/assets/images/network-topology-page.png" alt="List devices" width="800"/>
+
+## How to use this app
+This app is entirely depends on the backend app, so you need to deploy the backend app first, then you can deploy this app.
```

Comment:
Can you add some info or a link on how to deploy the backed app?

In addition, "this app" appears two time in the sentence: whad do you think to reword it a bit?

Reply:
I can't think about an alternative for the two times `this app`, what could be a good replacement for it?

Reply:
I can't think about an alternative for the two times `this app`, what could be a good replacement for it?

Reply:
but I added more links to the sentence to deploying backend and frontend app too

Reply:
> I can't think about an alternative for the two times this app, what could be a good replacement for it?

WDYT, something like "The backend app is the only requirement, so deploy the backend app and then you're ready to deploy this app." (feel free to reword it!)

Reply:
done